src/ui/common/src/components/workflows/SideSheets/OperatorResultsSideSheet.tsx

```diff
@@ -25,7 +25,6 @@ import {
     RelationalDBExtractParams,
     RelationalDBLoadParams
 } from "../../../utils/operators";
-import { CopyBlock } from "react-code-blocks";
```

Comment:
CopyBlock is giving us some trouble with building, so removing it for now.

We should revisit and find another component to use for the code blocks/copying in another task.

@vsreekanti  I noticed that there is a related PR here:
https://github.com/aqueducthq/enterprise/pull/521

Perhaps @Boyuan-Deng can take this on as some future work?